js/augurs-prophet-js/src/lib.rs

```diff
@@ -227,6 +235,61 @@ impl From<Seasonality> for augurs_prophet::Seasonality {
     }
 }
 
+/// Can be used to specify custom regressors
+#[derive(Clone, Debug, Default, Deserialize, Serialize, Tsify)]
+#[serde(rename_all = "camelCase")]
+#[tsify(from_wasm_abi, into_wasm_abi, type_prefix = "Prophet")]
+pub struct Regressor {
+    mode: FeatureMode,
+
+    #[tsify(optional, type = "number")]
+    prior_scale: Option<PositiveFloat>,
+
+    standardize: Standardize,
```

Comment:
It might be nice to make this optional in the JS API so users don't have to specify it, and we use the default if it's not provided.

```suggestion
    /// Whether to standardize this regressor.
    ///
    /// The default is to use automatic standardization, which will standardize
    /// numeric regressors and leave binary regressors alone.
    #[serde(default)]
    standardize: Standardize,
```